shared/kubernetes/utils.go

```diff
@@ -93,3 +95,29 @@ func GetPodInfosForPods(pods []core.Pod) []*shared.PodInfo {
 	}
 	return podInfos
 }
+
+
+func checkTappersCanStartOnNodes(kubernetesProvider *Provider, context context.Context, nodesToTap []string) {
+	var nodeNameToDetails = make(map[string]core.Node, 0)
+	if nodes, err := kubernetesProvider.GetNodes(context, ""); err == nil {
+		for _, item := range nodes.Items {
+			nodeNameToDetails[item.Name] = item
+		}
+	} else {
+		logger.Log.Debugf("cannot check nodes of cluster")
+	}
+
+	for _, item := range nodesToTap {
+		nodeDetails, ok := nodeNameToDetails[item]
+		if !ok {
+			logger.Log.Debugf("Cannot get this node details to check if the tapper can start on it")
```

Comment:
Lets add more details to the logs
```suggestion
			logger.Log.Debugf("Failed to find node to tap %v in cluster nodes %v", item, list(nodeNamesToDetails.keys))
```